scripts/demo_indexing.py

```diff
@@ -471,14 +471,27 @@ def via_rl(link=None):
     Algorithm(
         "Node2Vec",
         weighted=T(link="node-classification/node2vec-weighted-node-classification"),
-        nc=via_rl(link="node-classification/node2vec-node-classification"),
+        nc=[
+            T(
```

Comment:
Just to reiterate/make concrete, what we discussed on the stand-up, we could potentially collapse this like:

```suggestion
            T("via", details="via embedding vectors"),
            T(
```

and then remove the `via`/`via embedding vectors` from the later ones, to maybe make the text easier to read.